### PR TITLE
Fix 2185097: remove core electrons for an element only once (issue #42)

### PR DESCRIPTION
**Pull request by briling**

### CODE REVIEW

**Comment by YAY-C** [comment]
looks like the test-data is missing!

File: qstack/spahm/LB2020guess.py [hunk 1]
```diff
@@ -290,8 +290,13 @@ def use_charge(self, mol):
 
   def use_ecp(self, mol, acbasis):
       acbasis = copy.deepcopy(acbasis)
+      q_cleaned = set()
       for iat, z in enumerate(mol.atom_charges()):
           q = mol.atom_pure_symbol(iat)
+          if q in q_cleaned:
+              continue
+          else:
+              q_cleaned.add(q)
```
Comment (YAY-C) [inline qstack/spahm/LB2020guess.py:299]: where is `q_cleaned` used ?

File: tests/test_spahm.py [hunk 2]
```diff
@@ -36,6 +36,14 @@ def test_spahm_LB_ecp():
                        -0.07260264])
     assert np.allclose(R, true_R)
 
+    mol2 = compound.xyz_to_mol(path+'/data/I2.xyz', 'minao',
```
Comment (YAY-C) [inline tests/test_spahm.py:39]: file missing ?

**Review by YAY-C — CHANGES_REQUESTED** [review]

File: qstack/spahm/LB2020guess.py (hunk 1 above)
Comment (briling) [reply qstack/spahm/LB2020guess.py:299]: what do you mean sorry?

File: tests/test_spahm.py (hunk 2 above)
Comment (briling) [reply tests/test_spahm.py:39]: yeah also true_R2 is wrong!

File: qstack/spahm/LB2020guess.py (hunk 1 above)
Comment (YAY-C) [reply qstack/spahm/LB2020guess.py:299]: my bad, all good :ok_hand:

Comment (briling) [reply qstack/spahm/LB2020guess.py:299]: if `q` is already in `q_cleaned` then we dont remove core electrons

**Review by YAY-C — APPROVED** [review]

**Pull request merged**